### PR TITLE
Feat/711 show hits per day graph

### PR DESCRIPTION
**Pull request by leXuanNha**
This is the pull request integrate with API. Work with filter later.

### CODE REVIEW

File: packages/marketplace/src/components/ui/analytics/detailed/detailed-tab.tsx [hunk 1]
```diff
@@ -221,7 +226,34 @@ export const handleFetchAppUsageStatsDataUseEffect = (fetchAppUsageStatsData: ()
...
 }
 
-export const DetailedTab: React.FC<DetailedTabProps> = ({ installations, developer, appUsageStats, loadStats }) => {
+export const handleFetchHttpTrafficPerDayDataUseCallback = (
+  loadHttpTrafficPerDay?: (params: HttpTrafficPerDayParams) => void,
+) => {
+  return () => {
+    if (loadHttpTrafficPerDay) {
+      loadHttpTrafficPerDay({
+        applicationId: ['4fbbb1e8-bad0-43a2-98f9-bfb9bba366e7'],
```
Comment (willmcvay) [inline packages/marketplace/src/components/ui/analytics/detailed/detailed-tab.tsx:235]: Why are these dates and the applicationId hardcoded?

Comment (leXuanNha) [reply packages/marketplace/src/components/ui/analytics/detailed/detailed-tab.tsx:235]: It's temporary before I integrate with the filter.

Comment (leXuanNha) [reply packages/marketplace/src/components/ui/analytics/detailed/detailed-tab.tsx]: Fixed it.

File: packages/marketplace/src/components/ui/analytics/detailed/detailed-tab.tsx [hunk 2]
```diff
@@ -40,21 +43,32 @@ export const handleMapAppNameToInstallation = (
   })
 }
 
+const handleDefaultFilter = (developerAppDataArray: AppSummaryModel[]) => {
```
Comment (nphivu414) [inline packages/marketplace/src/components/ui/analytics/detailed/detailed-tab.tsx:46]: This function needs to be tested

File: packages/marketplace/src/components/ui/analytics/detailed/detailed-tab.tsx [hunk 3]
```diff
@@ -80,23 +94,48 @@ export const handleFetchAppUsageStatsDataUseEffect = (fetchAppUsageStatsData: ()
   }
 }
 
+export const handleFetchHttpTrafficPerDayDataUseCallback = (
```
Comment (nphivu414) [inline packages/marketplace/src/components/ui/analytics/detailed/detailed-tab.tsx:97]: This function needs to be tested

File: packages/marketplace/src/components/ui/analytics/detailed/__test__/detailed-tab.test.tsx [hunk 4]
```diff
@@ -91,6 +92,13 @@ describe('OverviewPage', () => {
           pageSize: 9999,
         }),
       )
+      expect(spyDispatch).toBeCalledWith(
+        httpTrafficPerDayRequestData({
+          applicationId: ['09043eb8-9e5e-4650-b7f1-f0cb62699027', '261da083-cee2-4f5c-a18f-8f9375f1f5af'],
```
Comment (nphivu414) [inline packages/marketplace/src/components/ui/analytics/detailed/__test__/detailed-tab.test.tsx:97]: Why do we expect `httpTrafficPerDayRequestData` action to be dispatched when `handleFetchAppUsageStatsDataUseCallback` function runs?
`httpTrafficPerDayRequestData` action should be dispached when `handleFetchHttpTrafficPerDayDataUseCallback` runs instead

**Pull request closed without merging**